iree/turbine/kernel/wave/expansion.py

```diff
@@ -553,6 +553,53 @@ def get_dim_scaling(
     return dim_scaling
 
 
+def _expand_mma_tiled_reduction(
+    mma: MMA,
+    trace: CapturedTrace,
+    dim_query: dict[IndexSymbol, int],
+    dim_scaling: dict[IndexSymbol, int],
+    context: ExpandedNodeMap,
+    get_node_dim_scaling: Callable[[fx.Node], dict[IndexSymbol, int]],
+    res_idx: int,
+) -> CustomOp:
+    latest_reduced_op = mma
+    for scale_idx in range(1, dim_scaling[mma.reduction_dim]):
```

Comment:
Just wondering why this has to start from 1?

Reply:
Added the doc here, it was from the original code
`The initial nodes are expanded in the first dimension, so we start from 1`